LondonBritishLibrary/orient/BLorient573.xml

```diff
@@ -31,21 +31,396 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         </altIdentifier>
                         <altIdentifier>
                             <idno>Wright 190</idno>
-                            
+
                         </altIdentifier>
-                        
+
                     </msIdentifier>
                     <msContents>
-                        <summary/>
-                        
-                        
-                    </msContents>
-                    
-                    
-                    
-                    
-                    
-                    
+
+<msItem xml:id="ms_i1">
+<locus from="3r"/>
+<title xml:lang="gez" type="complete" ref="LIT1850Malkea"/>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.1">
+<locus from="3r"/>
+<title xml:lang="gez" type="complete" ref="LIT1716Kidanz">Morning prayer</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.2">
+<locus from="11v"/>
+<title xml:lang="gez" type="complete" ref="LIT2444Temher">Tǝmhǝrta Ḫǝbuʾāt</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.3">
+<locus from="16r"/>
+<title xml:lang="gez" type="complete" ref="LIT2444Temher">Hymn</title>
+<incipit>ሰላም፡ ለሕማምከ፡ ቤተ፡ አይሁድ፡ ዘአንደደ፡ ወረሰዮሙ፡ ሐመደ።</incipit>
```

Comment:
here the xml:lang="gez" in the title is wrong, it is English. The same applies for many cases below